plugins/polio/models/base.py

```diff
@@ -1582,13 +1818,24 @@ class EarmarkedStockChoices(models.TextChoices):
         RETURNED = "returned", _("Returned")  #     3. Earmark -> Usable
 
     earmarked_stock_type = models.CharField(
-        max_length=20, choices=EarmarkedStockChoices.choices, default=EarmarkedStockChoices.CREATED
+        max_length=20,
+        choices=EarmarkedStockChoices.choices,
+        default=EarmarkedStockChoices.CREATED,
     )
-    vaccine_stock = models.ForeignKey(VaccineStock, on_delete=models.CASCADE, related_name="earmarked_stocks")
-    campaign = models.ForeignKey(Campaign, on_delete=models.CASCADE)
-    round = models.ForeignKey(Round, on_delete=models.CASCADE)
+    vaccine_stock = models.ForeignKey(
+        VaccineStock, on_delete=models.CASCADE, related_name="earmarked_stocks"
+    )
+    campaign = models.ForeignKey(
+        Campaign, on_delete=models.CASCADE, null=True, blank=True
+    )
+    temporary_campaign_name = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
I have to mention that we shouldn't have both `null=True` and `blank=True` for string fields as it creates 2 values for 'no data'. 

Django doc: https://docs.djangoproject.com/en/5.1/ref/models/fields/#field-options